src/db.h

```diff
@@ -141,6 +141,17 @@ enum data_kind {
 const char *
 db_data_kind_label(enum data_kind data_kind);
 
+
+/* Indicates user marked status on a track */
+enum flag_kind {
+  FLAG_KIND_NA = 0,     // unset
+  FLAG_KIND_DELETE,
+  FLAG_KIND_REXCODE,
+  FLAG_KIND_REVIEW,
```

Comment:
Maybe use bit field enumeration? That way they can be combined if it should ever be relevant.